src/lib.rs

```diff
@@ -319,7 +319,16 @@ impl<B: MysqlShim<W>, R: Read, W: Write> MysqlIntermediary<B, R, W> {
                                 w.completed(0, 0)?;
                             }
                         }
+                    } else if q.starts_with(b"USE ") || q.starts_with(b"use ") {
+                        let w = InitWriter {
+                            writer: &mut self.writer,
+                        };
+                        let schema = ::std::str::from_utf8(&q[3..])
+                            .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))?;
+                        let schema = schema.trim().strip_suffix(';').unwrap().replace("`", "");
```

Comment:
I don't think the `unwrap` will quite work here reading the docs on [`strip_suffix`](https://doc.rust-lang.org/std/primitive.str.html#method.strip_suffix). On second thought, I think what we want is:
```rust
let schema = schema.trim().trim_end_matches(';').trim_matches('`');
```

Reply:
That should also take care of the failing Minimum supported Rust version test.